packages/core/core-flows/src/payment-collection/steps/update-payment-collection.ts

```diff
@@ -18,6 +19,10 @@ export const updatePaymentCollectionStepId = "update-payment-collection"
 export const updatePaymentCollectionStep = createStep(
   updatePaymentCollectionStepId,
   async (data: StepInput, { container }) => {
+    if (!isPresent(data) || !isPresent(data.selector)) {
```

Comment:
note: main fix of the PR is here

Reply:
Just out of curiosity, what kind of updates were you seeing without the fix?

Reply:
Updates all the payments collections in the db with the params passed in

Reply:
Oh damn 😳